client/src/components/App.jsx

```diff
@@ -20,21 +22,23 @@ class App extends React.Component {
   // HTTP Request Handlers
   fetchReviews() {
     let url = `/api/models/${this.state.productcode}/reviews`;
-    fetch(url)
-      .then(response => response.json())
+    // fetch(url)
+    // .then(response => response.json()) 
+    axios(url)
```

Comment:
fetch and axios are so much cleaner and easier than ajax. I wonder why we ever did ajax